src/components/Stepper/type.ts

```diff
@@ -0,0 +1,7 @@
+import React, { ReactNode } from "react";
+
+export interface BaseStepperProps {
+  onAddClick: () => void;
+  onRemoveClick: () => void;
+  quantity?: number;
```

Comment:
alasannya buat 3 props ini apa hib, sama kegunaannya apa aja?

Reply:
- onAddClick : fungsi ketika plus icon di klik
- onRemoveClick: fungsi ketika minus icon di klik
- quantity: buat nampilin kuantitas yang ada (refering to figma design)

kenapa dibikin props karena gue mikir fungsi onAdd sm onRemove itu pasti dynamic penggunaannya, bisa jadi ada behavior lain selain increment state `quantity` kayak yang di commerce kemarin

<img width="178" alt="Screen Shot 2022-07-11 at 13 49 04" src="https://user-images.githubusercontent.com/104057020/178205130-4b600004-53d4-46c0-8b87-b0bc45f643fe.png">


<img width="366" alt="Screen Shot 2022-07-11 at 13 48 45" src="https://user-images.githubusercontent.com/104057020/178205087-5b344f38-9a8a-479d-9647-fddcd71a0231.png">

Reply:
mmm oke then i see. maybe instead of onAddClick dan onRemoveClick, jadinya onPlusClick, atau onMinusClick?

mikirnya sih karena kalo develop ngeliatnya sesuai design-nya, rada aneh gt kalo onAdd dan onRemove sedangkan ga ada direct utility yang nge indikasikan yang mana yang add sama yang mana yang remove